File: 2st/Yeonwoo/두 수의 연산값 비교하기.py
def solution(a, b):
    answer = 0
    str_a_b = str(a)+str(b)
    if 2*a*b > int(str_a_b):
        answer = 2*a*b
    elif 2*a*b == int(str_a_b):
        answer = str_a_b
    elif 2*a*b < int(str_a_b):
        answer = int(str_a_b)
    return answer

File: 4st/Jisoo/programmer/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        int d=1,t=b;
        while(t>0){
            d*=10;t/=10;
        }
        int ab=a*d+b,abb=2*a*b;
    
       
        return ab>=abb?ab:abb;
    }
}